src/transformers/__init__.py

```diff
@@ -401,6 +401,12 @@
         "TrOCRConfig",
         "TrOCRProcessor",
     ],
+    "models.tvlt": [
+        "TVLT_PRETRAINED_CONFIG_ARCHIVE_MAP",
+        "TvltConfig",
+        "TvltFeatureExtractor",
```

Comment:
Looks like the feature extractor relies on torchaudio, hence it will need to be placed below is_speech_available